src/sentry/uptime/endpoints/organization_uptime_alert_index_count.py

```diff
@@ -0,0 +1,66 @@
+from drf_spectacular.utils import extend_schema
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import NoProjects
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.constants import ObjectStatus
+from sentry.models.organization import Organization
+from sentry.uptime.models import ProjectUptimeSubscription
+from sentry.utils.auth import AuthenticatedHttpRequest
+
+
+@region_silo_endpoint
+@extend_schema(tags=["Uptime Monitors"])
+class OrganizationUptimeAlertIndexCountEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+    owner = ApiOwner.CRONS
+    permission_classes = (OrganizationPermission,)
+
+    def get(self, request: AuthenticatedHttpRequest, organization: Organization) -> Response:
+        """
+        Retrieves the count of uptime monitors for an organization.
+        """
+        try:
+            filter_params = self.get_filter_params(request, organization, date_filter_optional=True)
+        except NoProjects:
+            return self.respond(
+                {
+                    "counts": {
+                        "total": 0,
+                        "active": 0,
+                        "disabled": 0,
+                    },
+                }
+            )
+
+        queryset = ProjectUptimeSubscription.objects.filter(
+            project__organization_id=organization.id, project_id__in=filter_params["project_id"]
+        ).exclude(
+            status__in=[
+                ObjectStatus.PENDING_DELETION,
+                ObjectStatus.DELETION_IN_PROGRESS,
+            ]
+        )
```

Comment:
Could you actually have this pull from the `Detector` model instead? Here's where we create those from project subs

https://github.com/getsentry/sentry/blob/ad75d516fab4ffee7c7b06627b883738b8dd6bd2/src/sentry/uptime/models.py#L401-L412

Reply:
Yup! Changed to check that instead